pylib/cqlshlib/test/test_copyutil.py

```diff
@@ -75,42 +81,39 @@ def _test_get_ranges_murmur3_base(self, opts, expected_ranges):
         }
         # merge override options with standard options
         overridden_opts = dict(self.opts)
-        for k,v in opts.items():
+        for k, v in opts.items():
             overridden_opts[k] = v
         export_task = ExportTask(shell, self.ks, self.table, self.columns, self.fname, overridden_opts, self.protocol_version, self.config_file)
-        assert export_task.get_ranges() == expected_ranges
-
-    def test_get_ranges_murmur3(self):
-        """
-        Test behavior of ExportTask internal get_ranges function
-        """
+        export_ranges = export_task.get_ranges()
+        assert export_ranges == expected_ranges,\
+            "Expected: {e}\n Actual:{a}".format(e=expected_ranges, a=export_ranges)
 
+    def test_murmur3_get_ranges_invalid_input(self):
```

Comment:
I have decided to split into multiple test cases.
I have no idea why ppl prefer comments over correctly named methods.